keyboards/xbows/nature/readme.md

```diff
@@ -0,0 +1,22 @@
+# NATURE
+![NATURE](https://cdn.shopify.com/s/files/1/0014/7623/1237/files/xbows_1_720x_99574cc7-a547-4c1d-b847-482ec282f45b_540x.png)
+
+Nature, A 86 Keys Ergonomic Mechanical Keyboard with RGB backlight.
+If you spend your work days in front of a computer, you have likely felt some discomfort after a long day of typing. While spending less time on our computers would be the ideal remedy for this problem, that is not a realistic solution for most people. X-Bows was designed to limit the stress on your hands and wrists while also providing a productive and stylish typing experience.
+
+
+X-Bows was designed with three criteria in mind:
+*Enhancing the comfort and ergonomics of the keyboard
+*Limiting the learning curve for our new layout
+*Creating an attractive design
+
+
+Keyboard Maintainer: X-BOWS  
+Hardware Supported: X-BOWS Nature  
+Hardware Availability: [X-BOWS](https://x-bows.com/)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make xbows/nature:default
+
```

Comment:
Can you add "clear instructions on how to reset the board into bootloader mode" as per [the PR Checklist](https://docs.qmk.fm/#/pr_checklist?id=keyboard-prs)?

Reply:
I think this still needs to be added.

Reply:
Flashing example for this keyboard:

    make xbows/nature:default:flash

Is that it?   I added.